src/makemsix/main.cpp

```diff
@@ -331,6 +343,10 @@ int main(int argc, char* argv[])
                     [](State& state, const std::string&) { return state.SkipLanguage(); }),
                 Option("-sp", false, "Only for bundles. Skips matching packages with of the same system. By default unpacked application packages will only match the platform.",
                     [](State& state, const std::string&) { return state.SkipPlatform(); }),
+                Option("-sa", false, "Only for bundles. Skips all applicability checks for packages.",
+                    [](State& state, const std::string&) { return state.SkipApplicability(); }),
+                Option("-flat", false, "Only for bundles. Places unpacked packages side-by-side with the bundle folder. By default unpacked packages will be nested inside the bundle folder",
+	                [](State& state, const std::string&) { return state.UnpackWithFlatStructure(); }),
```

Comment:
>	 [](start = 0, length = 1)

Tab

Reply:
Still more in your changes

---
In reply to: [288787944](https://github.com/Microsoft/msix-packaging/pull/146#discussion_r288787944) [](ancestors = 288787944)